core/build_playerglobal/src/lib.rs

```diff
@@ -0,0 +1,104 @@
+//! An internal Ruffle utility to build our playerglobal
+//! `library.swf`
+
+use std::collections::HashSet;
+use std::fs::File;
+use std::path::PathBuf;
+use std::process::Command;
+use swf::DoAbc;
+use swf::Header;
+use swf::SwfStr;
+use swf::Tag;
+use walkdir::WalkDir;
+
+/// If successful, returns a list of paths that were used. If this is run
+/// from a build script, these paths should be printed with
+/// cargo:rerun-if-changed
+pub fn build_playerglobal(
+    repo_root: PathBuf,
+    out_dir: PathBuf,
+) -> Result<Vec<PathBuf>, Box<dyn std::error::Error>> {
+    let mut classes_dir = repo_root.clone();
+    classes_dir.push("core/src/avm2/globals/");
```

Comment:
nit: I think [`Path::join`](https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.join) can be used here.